code/src/Radio.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+export const Radio = ({ handleRadioInput }) => {
+  return <div className="question" tabIndex="0">
+      <fieldset>
+        <legend tabIndex="0">Which character has the best callsign?</legend>
+        <label for="mav" tabIndex="0">Maverick</label> 
+        <input type="radio" value="Maverick" id="mav" onChange={e => handleRadioInput(e.target.value)} required/>
```

Comment:
Very nice idea in general, but you forgot to add `checked` attribute. Because of that, I was able to check all 3 radio buttons (and I should select only one at the time)

Reply:
Fixed. I forgot to push where the radio buttons where in the same group or fieldset